tests/baseline.json

```diff
@@ -24,7 +24,7 @@
     "crude_birth_rate": 34.5,
     "total_fertility_rate_mean": 6.197679540983606,
     "asfr_mean": 111.25475,
-    "skyscrapers_mean": 2.0408163265306123,
+    "ageparity_mean": 2.0408163265306123,
     "spacing_stats_mean": Infinity,
```

Comment:
Any idea about the infinity here?

Reply:
Looks like it's been like that since [PR193](https://github.com/fpsim/fpsim/pull/193). I've made an issue to fix it [here](https://github.com/fpsim/fpsim/issues/303)

Reply:
I identified the root cause and wrote it up in [this ](https://github.com/fpsim/fpsim/issues/309)issue. @MObrien-IDM up to you on whether you'd like this addressed before/after merging this PR